bin/get_version_local.sh

```
@@ -0,0 +1,72 @@
+    formatted_clique+=$(transform_char "${clique:$i:1}")
+done
+
+# Combine the formatted string, add trailing carriage return
+formatted_result="$formatted_date$formatted_branch$formatted_commit$formatted_clique"
+formatted_result+="<&kp RET>"
+
+echo $formatted_result
+# Create new macro to define version, overwrite previous one
+
+echo '#define VERSION_MACRO' > "config/version.dtsi"
+echo 'macro_ver: macro_ver {' >> "config/version.dtsi"
```

Shellcheck (0.11.0)

[style] Consider using { cmd1; cmd2; } >> file instead of individual redirects.

(SC2129)